code/modules/client/preference/preferences_mysql.dm

```diff
@@ -38,7 +38,7 @@
 		toggles = text2num(query.item[7])
 		toggles2 = text2num(query.item[8])
 		sound = text2num(query.item[9])
-		volume = text2num(query.item[10])
+		volume_mixer = deserialize_volume_mixer(query.item[10])
```

Comment:
Failure to deserialize will just return the already present value. `initial()` does not work on lists.